spec/plainyearmonth.html

```diff
@@ -697,5 +652,41 @@ <h1>TemporalYearMonthToString ( _yearMonth_, _showCalendar_ )</h1>
         1. Return _result_.
       </emu-alg>
     </emu-clause>
+    <emu-clause id="sec-temporal-differencetemporalplainyearmonth" aoid="DifferenceTemporalPlainYearMonth">
+      <h1>
+        DifferenceTemporalPlainYearMonth (
+          _direction_: 1 or -1,
+          _yearMonth_: a Temporal.PlainYearMonth,
+          _other_: an ECMAScript language value,
+          _options_: an Object or *undefined*,
```

Comment:
```suggestion
          _options_: an ECMAScript language value,
```

Reply:
Why should we annotate options as an ECMAScript language value for some type but not the other types?